src/fides/api/alembic/migrations/versions/13403fee598a_table_updates_for_user_invite.py

```diff
@@ -0,0 +1,71 @@
+"""table updates for user invite
+
+Revision ID: 13403fee598a
+Revises: 5a8cee9c014c
+Create Date: 2024-01-06 00:05:49.159006
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from citext import CIText
+
+# revision identifiers, used by Alembic.
+revision = "13403fee598a"
+down_revision = "5a8cee9c014c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("create type disabledreason as enum('pending_invite')")
+    op.create_table(
+        "fides_user_invite",
+        sa.Column("id", sa.String(length=255), nullable=False),
+        sa.Column(
+            "created_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column("username", CIText(), nullable=False),
+        sa.Column("hashed_invite_code", sa.String(), nullable=False),
+        sa.Column("salt", sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint("id", "username"),
+    )
+    op.create_index(
+        op.f("ix_fides_user_invite_id"), "fides_user_invite", ["id"], unique=False
+    )
+    op.create_index(
+        op.f("ix_fides_user_invite_username"),
+        "fides_user_invite",
+        ["username"],
+        unique=False,
+    )
+    op.add_column("fidesuser", sa.Column("email_address", CIText(), nullable=True))
+    op.add_column("fidesuser", sa.Column("disabled", sa.Boolean(), nullable=False))
+    op.add_column(
+        "fidesuser",
+        sa.Column(
+            "disabled_reason",
+            sa.Enum("pending_invite", name="disabledreason"),
```

Comment:
would we prefer this to be a backend only enum?

Reply:
I tried doing that but I was having issues getting it to work, l'm ok with this for now